pkgs/native_assets_cli/lib/src/model/build_output.dart

```diff
@@ -33,8 +33,8 @@ class BuildOutput {
     Metadata? metadata,
   })  : timestamp = (timestamp ?? DateTime.now()).roundDownToSeconds(),
         assets = assets ?? [],
-        dependencies = dependencies ?? Dependencies([]),
-        metadata = metadata ?? Metadata({});
+        dependencies = dependencies ?? const Dependencies([]),
```

Comment:
It seems that this leads to unmodifiable lists trying to get modified:

https://github.com/dart-lang/native/actions/runs/6665614043/job/18115686857?pr=167